breadwallet/src/WalletManager+Auth.swift

```diff
@@ -56,10 +56,6 @@ extension WalletManager : WalletAuthenticator {
     static private var failedPins = [String]()
     
     convenience init(store: Store, dbPath: String? = nil) throws {
-        if !UIApplication.shared.isProtectedDataAvailable {
```

Comment:
The only code path that leads here is already inside a `guardProtected` call so this is not necessary and it is calling a UI API on background thread.